github-connector-demo/src_hd/connector/github/demo/HealthMonitor/HealthMonitor.xhtml

```diff
@@ -14,7 +14,7 @@
         
         <p:dataTable id="reposTable" var="repo" widgetVar="reposTable" value="#{healthMonitorBean.repos}" rows="10"
             paginatorTemplate="{RowsPerPageDropdown} {FirstPageLink} {PreviousPageLink} {CurrentPageReport} {NextPageLink} {LastPageLink}"
-            paginator="true" paginatorPosition="bottom" rowsPerPageTemplate="10,20,50">
+            paginator="true" paginatorPosition="bottom" rowsPerPageTemplate="10,20,50" lazy="false">
```

Comment:
auto added? do we want that?

Reply:
Yes, it was auto added. For now this table is not lazy so I think it should be fine if its set to false